install_config/aggregate_logging.adoc

```diff
@@ -3299,3 +3299,107 @@ link:https://github.com/Scribery/tlog[Tlog].
 |Scrubbed invalid output characters as bytes.
 |===
 endif::openshift-origin[]
+
+== Manual Elasticsearch rollouts
+
+The latest release of the Aggregated Logging stack has updated the Elasticsearch
+Deployment Config object so that it no longer has a Config Change Trigger meaning
+any changes to the DC will not result in an automatic rollout. This was to prevent
+unintended restarts happening in the ES cluster which could create lots of shard
+balancing as cluster members restart.
+
+[[elasticsearch-rolling-restart]]
+=== Performing an Elasticsearch rolling cluster restart
+
+A rolling restart is appropriate to be performed when not changing major versions
+of Elasticsearch. This will be the more commonly used restart policy.
+
+[NOTE]
+====
+Any action you do for a ES cluster will need to be repeated for the ops cluster
+if `openshift_logging_use_ops` was configured to be `True`.
+====
+
+1. Prevent shard balancing happening when we are bring down nodes on purpose
+From any ES pod in the cluster:
+```
+curl -s --cacert /etc/elasticsearch/secret/admin-ca \
+        --cert /etc/elasticsearch/secret/admin-cert \
+        --key /etc/elasticsearch/secret/admin-key \
+        -XPUT 'https://localhost:9200/_cluster/settings' \
+        -d '{ "transient": { "cluster.routing.allocation.enable" : "none" } }'
+```
+1. Waiting for completion between each one, for each DC you have for an ES cluster,
+use `oc rollout latest` to deploy the latest version of the DC object. You will see
+a new pod deployed. Once the pod has two ready containers, you can move on to the
+next DC.
+1. Once all DCs for the cluster have been rolled out, re-enable shard balancing.
+Again from any ES pod in the cluster:
+```
+curl -s --cacert /etc/elasticsearch/secret/admin-ca \
+        --cert /etc/elasticsearch/secret/admin-cert \
+        --key /etc/elasticsearch/secret/admin-key \
+        -XPUT 'https://localhost:9200/_cluster/settings' \
+        -d '{ "transient": { "cluster.routing.allocation.enable" : "all" } }'
+```
+
+[[elasticsearch-full-restart]]
+=== Performing an Elasticsearch full cluster restart
+
+A full restart is appropriate to be performed when changing major versions
+of Elasticsearch.
```

Comment:
"... or other changes which might put data integrity a risk during the change process."